lib/rucio/transfertool/fts3.py

```diff
@@ -356,44 +352,27 @@ def bulk_query(self, transfer_ids, timeout=None):
         :returns: Transfer status information as a dictionary.
         """
 
-        jobs = None
-
         if not isinstance(transfer_ids, list):
             transfer_ids = [transfer_ids]
 
-        responses = {}
-        fts_session = requests.Session()
+        """# multi-VO mode
```

Comment:
Don't leave commented code. If it's not needed, remove it

Reply:
Removed